tests/test_pinger.py

```diff
@@ -46,14 +46,14 @@ def test_read_status_invalid_json(self):
 
     def test_read_status_invalid_reply(self):
         self.pinger.connection.receive(
+            # no motd, see also #922
             bytearray.fromhex(
                 "4F004D7B22706C6179657273223A7B226D6178223A32302C226F6E6C696E65223A307D2C2276657273696F6E223A7B226E616"
                 "D65223A22312E382D70726531222C2270726F746F636F6C223A34347D7D"
             )
         )
 
-        with pytest.raises(IOError):
-            self.pinger.read_status()
+        self.pinger.read_status()
```

Comment:
We already had a test for #992. Interesting detail, `test_read_status_invalid_reply` in `test_async_pinger` calls `test_ping` instead `read_status`, is it a bug?

Reply:
Definitely some confusion on that naming. File being named `test_async_pinger.py` would imply that all those tests involve the pinger. Perhaps this was a copy/pasted test case whose name was not noticed during review?